hack/e2e.sh

```diff
@@ -13,24 +13,28 @@ fi
 
 if [[ -z "$SUITE" || "$SUITE" == "enduser" ]]; then
   echo "Running end user e2e tests"
-  # Login as osadmin to simulate a regular user
+  # Login as enduser to simulate a regular user
   password=$(awk '/^  endUserPasswd:/ { print $2 }' <_data/containerservice.yaml)
   fqdn=$(awk '/^  fqdn:/ { print $2 }' <_data/containerservice.yaml)
   export KUBECONFIG=_data/_out/enduser.kubeconfig
+  # oc login is going to create the enduser.kubeconfig for us
   oc login $fqdn --username enduser --password $password --insecure-skip-tls-verify=true
-  go test ./test/e2e -test.v -ginkgo.v -ginkgo.focus="\[EndUser\]" -ginkgo.noColor -ginkgo.randomizeAllSpecs -tags e2e -kubeconfig ../../_data/_out/enduser.kubeconfig "${ARTIFACT_FLAG:-}"
+  export KUBECONFIG_END_USER=../../_data/_out/enduser.kubeconfig
```

Comment:
Can you revert to the use of KUBECONFIG for end user and azure cluster reader? Is there a reason we are ditching KUBECONFIG altogether for these types of tests?

Reply:
I would like to have the `KUBECONFIG_`  usage consistent across the different types of tests. CustomerAdmin tests also require an enduser kubeconfig. Since CustomerAdmin requires 3 clients with 3 different kubeconfigs, those kubeconfigs can only be supplied for each client using the `KUBECONFIG_` variables. 

tldr; I didn't want to use `KUBECONFIG_END_USER` for an end user kubeconfig for `CustomerAdmin` tests and then go back to `KUBECONFIG` also for an end user kubeconfig in `EndUser` tests.